Tools/WinMLRunner/WinMLRunner.vcxproj

```diff
@@ -153,7 +154,7 @@
     </ClCompile>
     <Link>
       <SubSystem>Console</SubSystem>
-      <AdditionalDependencies>dxgi.lib;d3d12.lib;windowsapp.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>dxgi.lib;d3d12.lib;windowsapp.lib;mscoree.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
Why is mscoree.lib needed as a dependency? From my understanding this is related to .NET Framework